src/converters/number-compare-converters.js

```diff
@@ -14,23 +38,118 @@ function parseArgs(limit, trueVal, falseVal, valueToCompare, matchString) {
     }
 }
 module.exports = {
+    /**
+     * Convert the model variable to true, or other values passed to the converter, based on whether the model variable is greater than the limit.
+     *
+     * **Example**
+     *
+     *      <div>
+     *          <!-- displays true or the number of widgets -->
+     *          <span data-f-bind="widgets | greaterThan(50)"></span> 
+     *
+     *          <!-- displays custom text -->
+     *          <span data-f-bind="widgets | greaterThan(50, 'Congratulations!', 'Better luck next year')"></span>
+     *      </div>
+     *
+     * @param {Number} limit The reference value to compare the model variable against.
+     * @param {Number} trueVal (Optional) The format (value) to display if the model variable is greater than `limit`. If not included, the display is `true`.
+     * @param {Number} falseVal (Optional) The format (value) to display if the model variable is less than or equal to `limit`. If not included, the display is the value of the model variable.
+     * @return {Any} If the model variable is greater than `limit`, returns trueVal or `true`, otherwise returns falseVal or `false`.
+     */
     greaterThan: function (limit) {
         var args = parseArgs.apply(null, arguments);
         return Number(args.input) > Number(limit) ? args.trueVal : args.falseVal;
     },
+
+    /**
+     * Convert the model variable to true, or other values passed to the converter, based on whether the model variable is greater than or equal to the limit.
+     *
+     * **Example**
+     *
+     *      <div>
+     *          <!-- displays true or the number of widgets -->
+     *          <span data-f-bind="widgets | greaterThan(50)"></span> 
+     *
+     *          <!-- displays custom text -->
+     *          <span data-f-bind="widgets | greaterThan(50, 'Congratulations!', 'Better luck next year')"></span>
+     *      </div>
+     *
+     * @param {Number} limit The reference value to compare the model variable against.
+     * @param {Number} trueVal (Optional) The format (value) to display if the model variable is greater than or equal to `limit`. If not included, the display is `true`.
+     * @param {Number} falseVal (Optional) The format (value) to display if the model variable is less than `limit`. If not included, the display is the value of the model variable.
+     * @return {Any} If the model variable is greater than or equal to `limit`, returns trueVal or `true`, otherwise returns falseVal or the value of the model variable.
```

Comment:
returns falseval if provided or echoes the input (also for all other converters here)